### PR TITLE
Feat:"Added Input"

### PR DESCRIPTION
**Pull request by sadathshariff**
Added
1. Input Fields
2. Radio Buttons
3.  Checkbox
input

### CODE REVIEW

**Review by vivekmehto — COMMENTED** [review]
code looks good but one suggestion please add some preview for better understanding.

File: CSS/Components/input.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.input-text:focus {
+  border-color: var(--primary-color);
+}
+
+.error > input {
+  border: 1px solid red;
+}
+.error-msg {
+  color: red;
+}
```
Comment (shuntzhere) [inline CSS/Components/input.css]: Use variables for colors.

File: CSS/Components/input.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+.input-text:hover {
+  border-color: var(--primary-color);
+}
+.input-text:focus {
+  border-color: var(--primary-color);
+}
```
Comment (shuntzhere) [inline CSS/Components/input.css]: These two styles can be merged.

**Review by shuntzhere — COMMENTED** [review]
Suggested some minor changes and maintain constant spaces between different styles.

File: CSS/Components/input.css (hunk 2 above)
Comment (sadathshariff) [reply CSS/Components/input.css]: Have made the necessary changes can you look into it

**Pull request merged**